CONTRIBUTING.md

```diff
@@ -1,41 +1 @@
-# How to Contribute
-
-Contributions of all kinds are welcome!
-
-Please visit our [Community Forums](https://community.bitwarden.com/) for general community discussion and the development roadmap.
-
-Here is how you can get involved:
-
-* **Request a new feature:** Go to the [Feature Requests category](https://community.bitwarden.com/c/feature-requests/) of the Community Forums. Please search existing feature requests before making a new one
-  
-* **Write code for a new feature:** Make a new post in the [Github Contributions category](https://community.bitwarden.com/c/github-contributions/) of the Community Forums. Include a description of your proposed contribution, screeshots, and links to any relevant feature requests. This helps get feedback from the community and Bitwarden team members before you start writing code
-  
-* **Report a bug or submit a bugfix:** Use Github issues and pull requests
-  
-* **Write documentation:** Submit a pull request to the [Bitwarden help repository](https://github.com/bitwarden/help)
-  
-* **Help other users:** Go to the [User-to-User Support category](https://community.bitwarden.com/c/support/) on the Community Forums
-  
-* **Translate:** See the localization (l10n) section below
-
-## Contributor Agreement
-
-Please sign the [Contributor Agreement](https://cla-assistant.io/bitwarden/desktop) if you intend on contributing to any Github repository. Pull requests cannot be accepted and merged unless the author has signed the Contributor Agreement.
-
-## Pull Request Guidelines
-
-* use `npm run lint` and fix any linting suggestions before submitting a pull request
-* commit any pull requests against the `master` branch
-* include a link to your Community Forums post
-
-# Localization (l10n)
-
-[![Crowdin](https://d322cqt584bo4o.cloudfront.net/bitwarden-desktop/localized.svg)](https://crowdin.com/project/bitwarden-desktop)
-
-We use a translation tool called [Crowdin](https://crowdin.com) to help manage our localization efforts across many different languages.
-
-If you are interested in helping translate the Bitwarden desktop app into another language (or make a translation correction), please register an account at Crowdin and join our project here: https://crowdin.com/project/bitwarden-desktop
-
-If the language that you are interested in translating is not already listed, create a new account on Crowdin, join the project, and contact the project owner (https://crowdin.com/profile/kspearrin).
-
-You can read Crowdin's getting started guide for translators here: https://support.crowdin.com/crowdin-intro/
+Please see https://github.com/cozy/cozy-stack/blob/master/docs/CONTRIBUTING.md
```

Comment:
I think we should create a `contributing` guide in the "front documentation". I think that stack is a bit different

Reply:
Where should we store this documentation? Maybe in cozy-ui? Or in a dedicated "cozy" repository?

Reply:
https://github.com/cozy/cozy.github.io/tree/dev/src/howTos/dev

Maybe we should create a `contritubing-to-frond-end.md` there? But we can remove this `contributing` file in the meantime?